core/src/main/scala/kafka/server/KafkaRequestHandler.scala

```diff
@@ -319,11 +349,16 @@ class BrokerTopicMetrics(name: Option[String], configOpt: java.util.Optional[Kaf
         RemoteStorageMetrics.FAILED_REMOTE_FETCH_PER_SEC_METRIC.getName -> MeterWrapper(RemoteStorageMetrics.FAILED_REMOTE_FETCH_PER_SEC_METRIC.getName, "requests"),
         RemoteStorageMetrics.FAILED_REMOTE_COPY_PER_SEC_METRIC.getName -> MeterWrapper(RemoteStorageMetrics.FAILED_REMOTE_COPY_PER_SEC_METRIC.getName, "requests")
       ).asJava)
+      metricGaugeTypeMap.putAll(Map(
+        BrokerTopicStats.RemoteCopyLagBytes -> GaugeWrapper(BrokerTopicStats.RemoteCopyLagBytes, new BrokerTopicAggregatedMetric)
+      ).asJava)
```

Comment:
```suggestion
        metricGaugeTypeMap.put(
            BrokerTopicStats.RemoteCopyLagBytes,
            GaugeWrapper(BrokerTopicStats.RemoteCopyLagBytes, new BrokerTopicAggregatedMetric))
```

Can we move the constant `RemoteCopyLagBytes` from `BrokerTopicStats` to `RemoteStorageMetrics`?

Reply:
Oki